part2.asm

```diff
@@ -0,0 +1,58 @@
+# COMP3410 Program Template
+# Author: Aaron Marshall
+# Assignment: PA[1] part 2
+# Date: 2/10/14
+# takes two inputted number and adds, subs, and mults
+	.data
+input: .asciiz "Input two integers.\n"
+subtext: .asciiz "\nSubtracting......\n"
+addtext: .asciiz "\nAdding...........\n"
+multitext: .asciiz "\nMultiplying......\n"
+goodbye: .asciiz "\nBye..............\n"
+
```

Comment:
-4 Instructions require a printed introduction including name, title, and purpose of program